presto-main/src/test/java/com/facebook/presto/sql/query/TestDistinctAggregations.java

```diff
@@ -272,4 +272,14 @@ public void testMixedDistinctWithFilter()
                         "FROM (VALUES (1, 3), (2, 4), (2, 4), (4, 5)) t (x, y)",
                 "VALUES (BIGINT '0', CAST(NULL AS BIGINT))");
     }
+
+    @Test
+    public void testUuidDistinct()
```

Comment:
@BryanCutler : We should check how this works in Prestissimo as well. Don't anticipate a problem, but it would be good to make it part of the tests.